src/views/AlertMessageView.js

```diff
@@ -11,15 +11,23 @@ class AlertMessageView extends Backbone.View {
 
     initialize(options) {
         this.mediator = options.mediator;
-        this.bindEvents(this.mediator);
+        this.bindEvents();
         this.message = {
             title: '',
             content: ''
         };
     }
 
-    bindEvents(mediator) {
-        mediator.on('app:alert', this.showAlert, this);
+    bindEvents() {
+        if (this.mediator === undefined || this.mediator === null) {
+            return;
+        }
+        this.mediator.on('app:alert', this.showAlert, this);
+    }
+
+    setMediator(mediator) {
```

Comment:
Is this only used by tests? If so, I think it would be better to refactor the classes in question to have constructors (rather than `initialize` function) so the mediator can be handed off when the class is created.